src/components/find-on-shelf/FindOnShelfManifestationListItem.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import { FC } from "react";
+import { useText } from "../../core/utils/text";
+import { Manifestation } from "../../core/utils/types/entities";
+
+export interface FindOnShelfManifestationListItemProps {
+  shelfmark: Manifestation["shelfmark"];
+  department: string | undefined;
+  location: string | undefined;
+  sublocation: string | undefined;
+  title: string;
+  publicationYear: string;
+  numberAvailable: number;
+}
+
+const FindOnShelfManifestationListItem: FC<
+  FindOnShelfManifestationListItemProps
+> = ({
+  shelfmark,
+  department,
+  location,
+  sublocation,
+  title,
+  publicationYear,
+  numberAvailable
+}) => {
+  const t = useText();
+
+  const shelfmarkString = shelfmark
```

Comment:
A variable that is called a string which can be undefined does not make so much sense I think

Reply:
That's a great point! :) Thx ❤️